biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/resolve/BndrunContainer.java

```diff
@@ -209,6 +215,20 @@ public FileSetRepository getFileSetRepository() throws Exception {
 		return fileSetRepository = dependencyResolver.getFileSetRepository(name, bundles, useMavenDependencies);
 	}
 
+	private void getEEfromBuild(Bndrun run) {
+		String runee = run.getProperty(Constants.RUNEE);
+
+		if (runee == null) {
+			Optional.ofNullable(project.getPlugin("org.apache.maven.plugins:maven-compiler-plugin"))
+				.map(Plugin::getConfiguration)
+				.map(Xpp3Dom.class::cast)
+				.map(dom -> dom.getChild("target"))
```

Comment:
I assume the target element is set to the actual value event when using maven properties or letting things default to the current JDK being run?

Reply:
yes, that's why I get it from here rather than the property, it may be set either way, but this holds the final result.